be/src/pipeline/exec/aggregation_sink_operator.h

```diff
@@ -39,7 +39,7 @@ class AggSinkOperatorBuilder final : public OperatorBuilder<vectorized::Aggregat
     bool is_sink() const override { return true; }
 };
 
-class AggSinkOperator final : public StreamingOperator<AggSinkOperatorBuilder> {
```

Comment:
If the operator does not depend on operator builder, then should remove all builders.